pkg/storaged/crypto-keyslots.jsx

```diff
@@ -511,20 +510,11 @@ function add_dialog(client, block) {
                                ]
                            }),
             Skip("medskip"),
-            PassInput("new_passphrase", _("New passphrase"),
-                      {
-                          visible: vals => !client.features.clevis || vals.type == "luks-passphrase",
-                          validate: val => !val.length && _("Passphrase cannot be empty"),
-                      }),
-            PassInput("new_passphrase2", _("Repeat passphrase"),
-                      {
-                          visible: vals => !client.features.clevis || vals.type == "luks-passphrase",
-                          validate: (val, vals) => {
-                              return (vals.new_passphrase.length &&
-                                                        vals.new_passphrase != val &&
-                                                        _("Passphrases do not match"));
-                          }
-                      }),
+            NewPassInput("new_passphrase", _("New passphrase"),
+                         {
+                             visible: vals => !client.features.clevis || vals.type == "luks-passphrase",
+                             validate: val => !val.length && _("Passphrase cannot be empty"),
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18906-20230609-093742-36fdfe4a-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)